### ISSUE TITLE
SFM: Produces only two cameras for some datasets

### ISSUE THREAD

**Issue by davll** [simonfuhrmann/mve#144]
`sfmrecon` used to work well with some data, but now it produces only TWO cameras (as initial pair).
I did not exploit how the bug happens, forking the repo instead.

[dataset link](https://www.dropbox.com/s/75ekpwuxs6yx8hx/e100vs.zip?dl=0)

**Comment by simonfuhrmann** [comment]
Hi David,

this is a known issue. SfM sometimes fails to select a proper initial pair. An easy fix is to use the `--initial-pair` command line option to `sfmrecon`  and select an initial pair yourself. If you have enough baseline, the problem should disappear.

Simon

**Comment by davll** [comment]
I tried `--initial-pair`, `--shared-intrinsics`, `--track-error-thres`, `--track-thres-factor` but it doesn't work.

**Comment by flanggut** [comment]
Which pair did you try for the initial pair?

**Comment by KevinCain** [comment]
I've noticed, over the course of several data sets, that 'sfmrecon' *does* makes reasonable choices for the initial pair in a sequence.  However, often in my limited experience even though there is more than adequate shared baseline between the auto-selected image pair, 'sfmrecon' returns only two cameras, as noted.

I have found that invoking 'sfmrecon' with the '--initial-pair' command line option nearly always returns output for all cameras, no matter the image pair specified by the user.
